spark-bigquery-pushdown/pushdown_common_src/main/scala/com/google/cloud/spark/bigquery/pushdowns/SparkExpressionConverter.scala

```diff
@@ -290,6 +291,62 @@ trait SparkExpressionConverter {
     })
   }
 
+  def convertWindowExpressions(expression: Expression, fields: Seq[Attribute]): Option[BigQuerySQLStatement] = {
+    Option(x = expression match {
+      case WindowExpression(windowFunction, windowSpec) =>
+        windowFunction match {
+          /**
+           * Since we can't use a window frame clause with navigation functions and numbering functions,
+           * we set the useWindowFrame to false
+           */
+          case _: Rank | _: DenseRank | _: PercentRank | _: RowNumber | _ =>
```

Comment:
Need to remove the last `_` otherwise the statement `case _` will never be executed

Reply:
My bad, addressed